static/js/redux/ui/modals/PeerModal.tsx

```diff
@@ -0,0 +1,281 @@
+        <h4>Study Buddies, Delivered</h4>
+        <p className="description">
+          See who your classmates are this semester! Click below to find Semester.ly
+          users in your courses, message them, or add them on Facebook!
+          <i>
+            By accepting this permission, any Semester.ly students in your courses will
+            be able to view your name and public Facebook profile.
+          </i>
+        </p>
+        <button className="lure-accept" onClick={optInClick}>
+          Yes, I&#39;m In
+        </button>
+      </div>
+    </div>
+  );
+
+  const height =
+    peers[0] && peers[0].shared_courses ? peers[0].shared_courses.length * 30 + 95 : 0;
+  const peerCards = peers.map((p) => {
+    const isFriend = p.is_friend ? (
+      <p className="friend-status">
+        <i className="fa fa-check" /> Friends
+      </p>
+    ) : null;
+    const sharedCourses = p.shared_courses.map((sc) => {
+      const colourIndex = courseToColourIndex[sc.course.id] || 0;
+      const inSection = sc.in_section ? <i className="fa fa-check" /> : null;
+      return (
+        <div className="shared-course" key={String(sc.course.id) + p.profile_url}>
+          <div
+            className="course-color-circle"
+            style={{ backgroundColor: COLOUR_DATA[colourIndex].background }}
+          >
+            {inSection}
+          </div>
+          <p className="course-title">{`${sc.course.code} - ${sc.course.name}`}</p>
+        </div>
+      );
+    });
+    return (
+      <div className="peer-card" key={p.profile_url}>
+        <div className="peer-card-wrapper" style={{ height }}>
+          <div className="card-hat">
+            <div
+              className="peer-pic"
+              style={{ backgroundImage: `url(${p.large_img})` }}
+            />
+            <div className="user-info">
+              <h3>{p.name}</h3>
+              <a href={p.profile_url} target="_blank" rel="noopener noreferrer">
+                <button className="view-profile-btn">
+                  <i className="fa fa-facebook-square" />
+                  View Profile
+                </button>
+              </a>
+              {isFriend}
+            </div>
+          </div>
+          <div className="shared-courses">{sharedCourses}</div>
+        </div>
+      </div>
+    );
+  });
+
+  const display = !isLoading ? (
+    <div className="main-modal-wrapper">
+      <div className="pm-header">
+        <h4>Your Classmates</h4>
+        <div className="key">
+          <div className="key-entry">
+            <div className="course-color-circle" style={{ backgroundColor: "#ddd" }}>
+              <i className="fa fa-check" />
+            </div>
+            <p>peer is in your class & section</p>
+          </div>
+          <div className="key-entry">
+            <div className="course-color-circle" style={{ backgroundColor: "#ddd" }} />
+            <p>peer is in your class only</p>
+          </div>
```

Comment:
Do you know why these are static icons instead of clickable/toggleable?

Reply:
I think it's because that's just the key to show what it means. They likely never had support for filtering it by class vs class & section